### DISCUSSION TITLE
Can add more doors in version 1.7.10?

### DISCUSSION THREAD

**Discussion by gegedelanjiaohoada** [HbmMods/Hbm-s-Nuclear-Tech-GIT#710] [Q&A]
Can those doors newly added to version 1.12.2 also be added to version 1.7.10?

**Comment by Mooshroom8** [comment]
Yeah. I really like them. The fire door is in 1.7.10 though, so maybe more will come!

**Comment by OsakaLinuxScraper** [comment]
1.12.2 is not managed by HBM.
Please put this comment on the respective link, not this version. The Respective link is [This](https://github.com/TheOriginalGolem/Hbm-s-Nuclear-Tech-GIT/issues)

_This was a bot, and this action was perfomed automatically, if you find this was inappropriate, please respond by saying "badbot", if you want to opt out, say "nobot"_
